src/ensembl/src/shared/components/external-link/ExternalLink.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { ReactComponent as LinkIcon } from 'static/img/shared/icon-xlink.svg';
+
+import styles from './ExternalLink.scss';
+
+export type ExternalLinkProps = {
+  label?: string;
```

Comment:
I don't think that the label or the container for label and link should be concerns of the ExternalLink component.
The only thing that the external link should do is decorate what would have otherwise been a regular link with an icon.
Its api should probably be as follows:
```
<ExternalLink href="path" classNames={{link, icon}}>some text</ExternalLink>
```
One possibility to consider is whether the icon is always rendered to the left, or whether we need a prop to position the icon to the left or to the right of the link text.

Compare the interface of react-router's Link component as an example:
```
<Link to="/about">About</Link>
```

Reply:
I have changed `linkUrl` to be `href`. 

The label seems to be appearing the same in most of the places where this component is used. It doesn't add much complexity either. Having it as a part of this component will make it easier to do some global changes. I will check with Andrea if this component is going to have many variations.

Reply:
The label (or rather name) is a part of the missing `ExternalReference` component, not `ExternalLink` 😃 
By the way, how will this case from pdf be handled?

![image](https://user-images.githubusercontent.com/6834224/77918592-00100e80-7294-11ea-8ca5-31d70f9a55fc.png)

Reply:
Speaking to Andrea, she mentioned that this component will always look the same:

```
Label -> Icon -> Link
or 
Icon -> Link
```

I guess I can rename it to `ExternalReference` in that case?

Regarding the special case, I will make some changes to handle it.

Reply:
> I guess I can rename it to ExternalReference in that case?

I think ExternalLink (a link plus an icon) is a valuable component. It can be used as part of a larger ExternalReference component that will know what to do with labels and, possibly, with multiple links per reference.

Reply:
Do we really need a larger ExternalReference component? I don't think there will be many cases like the one mentioned above. 

The most common use case of this component is going to be `Label -> Icon -> Link` .  The link text (usually some ID) wouldn't make sense on its own without the label in many places.

Reply:
> The most common use case of this component is going to be Label -> Icon -> Link . The link text (usually some ID) wouldn't make sense on its own without the label in many places.

We have examples on the current site of external links, e.g. this on the home page:
![image](https://user-images.githubusercontent.com/6834224/77957299-31f09780-72cb-11ea-9930-31933960ad0e.png)

or this on the [Help and docs page](https://www.ensembl.org/info/index.html):

![image](https://user-images.githubusercontent.com/6834224/77957455-66fcea00-72cb-11ea-8178-52d0662386f6.png)

Special styling of external links is a very common pattern, achievable even through pure CSS.

Reply:
Are we okay to leave this component as it is as Andrea mentioned that it will be widely used?

Reply:
I thought Andrea's comment:
> it will be used over and over again globally and often - so a component in itself

meant that the link without the label will be used over and over again, so is a component in itself. Did you read it differently?

Reply:
One potential way to avoid the ExternalLink component is to target external links with CSS. If all external links start with http, for example, this can be achieved by:

```
a[href^="http://"]:before,
a[href^="https://"]:before {
    content: "";
    display: inline-block;
    ...
    background-image: <path-to-svg> ;
}
```
we will probably have to do the above on help and docs pages, whose content we do not control, anyway. Don't know if it is a good idea to do this throughout.

Reply:
Created the `ExternalLink` component as discussed.